tests/conftest.py

```diff
@@ -9,6 +9,9 @@ def stub_environment():
     stubbed_environment_variables = [
```

Comment:
Probably out of scope but (while this is cool) the suggested way of doing this is with `unittest.mock.patch.dict`
https://stackoverflow.com/a/51754362
https://docs.python.org/3/library/unittest.mock.html#patch-dict

Reply:
Nice catch. I'd rather we do this than use the actual `os` override anyways.